tests/APIv2/UsersTest.php

```diff
@@ -129,6 +129,37 @@ public function testDeletePhoto() {
         $this->assertStringEndsWith('/applications/dashboard/design/images/defaulticon.png', $user['photoUrl']);
     }
 
+    /**
+     * Test confirm email is successful
+     */
+    public function testConfirmEmailSucceed() {
+        $emailKey = ['emailKey' => 'test123'];
+        $user = $this->testPost();
+
+        /** @var \UserModel $userModel */
+        $userModel = self::container()->get('UserModel');
+        $userModel->saveAttribute($user['userID'], 'EmailKey', 'test123');
+
+        $response = $this->api()->post("{$this->baseUrl}/{$user['userID']}/confirmEmail", $emailKey);
```

Comment:
The original issue stated the URL should look more like: /users/:id/confirm-email

You shouldn't have to change anything, but make sure you're testing with that URL.